CMakeLists.txt

```diff
@@ -285,10 +286,15 @@ link_directories(
 add_subdirectory(src)
 
 if (COVERAGE_ENABLE AND CMAKE_COMPILER_IS_GNUCXX)
-  # FIXME(aceforeverd): api_server_test failed to run on debug mode, skip temporarily
-  SETUP_TARGET_FOR_COVERAGE_LCOV(NAME coverage
-    EXECUTABLE ctest --force-new-ctest-process --output-on-failure -E "api_server_test"
-    DEPENDENCIES ${test_list})
+  if (COVERAGE_NO_DEPS)
+        SETUP_TARGET_FOR_COVERAGE_LCOV(NAME coverage
+        EXECUTABLE ctest --force-new-ctest-process --output-on-failure -E "api_server_test")
```

Comment:
by design, `make coverage-cpp` only runs `coverage` target and should pull dependent targets `coverage` job depends.

Reply:
But we delete files in build, if we run make again, it'll build again. No deps can ignore it.
So it's better to add a new cmd? e.g. `make coverage-cpp-no-deps`? User should use the original one.